core/monitoring/user-events/src/test/scala/org/apache/openwhisk/core/monitoring/metrics/KafkaSpecBase.scala

```diff
@@ -30,11 +30,25 @@ abstract class KafkaSpecBase
     with ScalaFutures
     with FlatSpecLike
     with EmbeddedKafka
-    with EmbeddedKafkaLike
     with IntegrationPatience
     with Eventually
     with EventsTestHelper { this: Suite =>
-  implicit val timeoutConfig: PatienceConfig = PatienceConfig(1.minute)
+  implicit val timeoutConfig: PatienceConfig = PatienceConfig(5.minute)
   override val sleepAfterProduce: FiniteDuration = 10.seconds
-  override protected val topicCreationTimeout = 60.seconds
+  override protected val topicCreationTimeout = 120.seconds
```

Comment:
I am just curious if this is to apply different timeouts for topic creation and publishing timeout or if there was any case that topic creation takes more than 60s.

Reply:
I've reverted the timeout changes I think it was just leftover from me trying to get the tests to work, but still think the topic creation needs to be the same timeout as produce timeout rather than the default of the test library which is 10s.

also I've added the cve's to the description. for bug fixes it would be hard to cover here rather than just pointing to the kafka release notes between 2.4.0 and 2.8.2.

Reply:
Thanks!